samples/endpoints/shell/src/AppRouter.tsx

```diff
@@ -119,8 +140,14 @@ export function AppRouter({ waitForMsw, sessionManager, telemetryService }: AppR
                         waitForMsw={waitForMsw}
                         onLoadPublicData={handleLoadPublicData}
                         onLoadProtectedData={handleLoadProtectedData}
+                        isPublicDataLoaded={!!featureFlags}
```

Comment:
All codebase combined, i think it's the first time i've seen you write the truthy/falsy synthax 😂